app/models/my_module.rb

```diff
@@ -205,6 +205,15 @@ def repository_rows_count(repository)
                              .count
   end
 
+  def assigned_repositories_list
```

Comment:
Does it return collection of Repository models?

Reply:
Updated